audiomate/corpus/conversion/base.py

```diff
@@ -49,9 +50,10 @@ def convert(self, corpus, target_audio_path):
         out_corpus = audiomate.Corpus()
         files_to_convert = []
 
-        print('Find utterances to convert ...')
-
-        for utterance in tqdm(corpus.utterances.values(), total=corpus.num_utterances):
+        for utterance in logger.progress(
```

Comment:
I'm unsure about that. It's nice if you're happy with textual output, but it's not comparable to tqdm. My idea is to provide some sort of listener callback that people could hook into. Does not require a lot of additional code I think.

Reply:
I completely agree with the callback approach.
But, I wanted to have some easy way to get some info about the progress without having to implement the callback on the user side.
My idea would be to provide a callback solution additionally.